pkgs/desktops/cinnamon/cinnamon-common/use-sane-install-dir.patch

```diff
@@ -0,0 +1,33 @@
+From f7e802959d7a5c217ed574cab30404fc769f174d Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Maciej=20Kr=C3=BCger?= <mkg20001@gmail.com>
+Date: Sat, 6 Feb 2021 14:26:26 +0100
+Subject: [PATCH] use sane install dir
```

Comment:
this patch checks out with me, feel free to PR. Motivation is that the correct usage of meson is to install into directories using their built-in options https://mesonbuild.com/Builtin-options.html#page-description. Or maybe there's an issue with debian having those per platform `/usr/` directories, can't remember.

Reply:
prefix is /usr/local sometimes, so not sure if this is something upstream wants, as they expect it in /usr

Reply:
it could also do something like `if (get_option('prefix') == '/usr/local')` or whatever

Reply:
not sure... most likely upstream won't like it either way. somebody else should feel free to try.